tests/unit/BusinessTypeConfirm.spec.ts

```diff
@@ -0,0 +1,88 @@
+import Vue from 'vue'
+import flushPromises from 'flush-promises'
+import { Wrapper } from '@vue/test-utils'
+import { getLastEvent } from '../get-last-event'
+import { wrapperFactory } from '../jest-wrapper-factory'
+import BusinessTypeConfirm from '@/components/Registration/BusinessTypeConfirm.vue'
+
+const validEvent = 'valid'
+/**
+ * @param BusinessTypeConfirm the value to pass to the component for the checkbox. The default value is "undefined". *
+ */
```

Comment:
If this JSDoc comment is obsolete, please delete it.

Reply:
Hey, don't resolve conversations that I created. Only resolve those that you created.

Reply:
Oh. Sorry.